nncf/openvino/graph/node_utils.py

```diff
@@ -49,6 +51,20 @@ def is_node_with_bias(node: NNCFNode, nncf_graph: NNCFGraph) -> bool:
     return bias_constant is not None
 
 
+def has_if_op(model: ov.Model) -> bool:
```

Comment:
I suggest using the name `is_ifmodel()` for this method.

Reply:
Updated to a new method